timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -66,6 +64,7 @@
     NPTS=NPTSModel,
     Theta=ThetaModel,
     ETS=ETSModel,
+    ARIMA=ARIMAModel,
```

Comment:
We need to assign a priority score to ARIMA as well.

Reply:
Looks like it was already there.